### DISCUSSION TITLE
Can I get statistical data on rides?

### DISCUSSION THREAD

**Discussion by karma-ashutosh** [nammayatri/nammayatri#456] [Q&A] [answered]
Hi,
I'm applying in Namma Yatri Hackathon. We're trying to propose a solution for map replacement and need some statistical info. Will it be possible to get some help here? 

I could not find any slack / Discord link, hence posting here.

**Comment by sidsethujuspay** [comment, marked as answer]
@karma-ashutosh What kind of info are you looking for? If it is ride related information, you can take a look at our [open data dashboard.](https://nammayatri.in/open/)
